tests/db/test_controls_workflows_mapping.py

```diff
@@ -0,0 +1,163 @@
+import pytest
+from sqlmodel import Session, SQLModel, create_engine
+from admyral.db.schemas.control_schemas import (
+    ControlSchema,
+    ControlsWorkflowsMappingSchema,
+)
+from admyral.db.schemas.workflow_schemas import WorkflowSchema
+from admyral.db.schemas.auth_schemas import UserSchema
+from admyral.db.schemas.workflow_control_results_schemas import (
+    WorkflowControlResultsSchema,
+)
+from sqlalchemy import text
+
+# Use SQLite for testing
+DATABASE_URL = "sqlite:///:memory:"
```

Comment:
The unit tests already expects that our postgres DB is running. I would suggest to use the postgres instead of sqlite. We should then also generalize the session fixture to use the same one across tests. See `tests/workers/conftest.py` for the fixture used by the other tests.